### ISSUE TITLE
Script Error

### ISSUE THREAD

**Issue by Gernatch** [iam4x/pokemongo-webspoof#163]
![screen shot 2016-07-21 at 10 14 43 pm](https://cloud.githubusercontent.com/assets/7558344/17044714/94b68170-4f90-11e6-9c8d-1b6337bc5771.png)

**Comment by Gernatch** [comment]
Oh... Didn't mention. Xcode 8. I bet that is the problem. It seems like the names of the UI elements may have changed.

**Comment by AbelIngrand** [comment]
If you go by yourself in Debug > Simulate location, do you see pokemonLocation ?

**Comment by dickverbunt** [comment]
Have the same issue with Xcode 7.3.1. I do see the menu item pokemonLocation.

It couldn't find the item. Going the the "Simulate Location" menu works. Also tried setting menu item 1 but also didn't work.

**Comment by Gernatch** [comment]
I originally thought it was a script issue. However I never made a single change to the script to cause it to work. I'm puzzled. Hopefully I will have time to mess with it this evening. 

Did you try disabling he randomization of movement? Just curious.

**Comment by Gernatch** [comment]
Yes. It works fine when clicking it. This is how I was playing for a while. I would move, then manually click it to actualize my move. 

For some reason it started working last night. The "only thing" I did was disable the randomization of my movements. It worked immediate after. Not sure yet if that's what fixed it or if it was just coincidence. 

Also noticed if you make the map full screen, it doesn't work. That makes sense since the Xcode UI isn't visible. Just a side note in case someone tries this. Not a bug. It's just how Apple Script works I do believe.

**Comment by dickverbunt** [comment]
Tried disabling the randomization. But didn't work.
If you isolate the script and try to run it it's not working. You get the same error.

So it doesn't have anything to do with the randomization I believe. But still strange that it started to work for your.

**Comment by Gernatch** [comment]
That's right. I did isolate the script too in my troubleshooting. Not sure yet how it started working. Will try working on it later since I have the best of both worlds.

**Comment by zekkd** [comment]
I got the same problem.

iOS 9
El Capitain

**Comment by Gernatch** [comment]
I've resolved the issue for myself. 3 simple steps. 

1) Make sure to **add Script Editor** under Security and Privacy > Privacy > (Unlock - lower left) > Browse to Script Editor (Under Applications > Utilities). Make sure it is checked.
![screen shot 2016-07-22 at 10 17 14 pm](https://cloud.githubusercontent.com/assets/7558344/17075201/10e8356c-505a-11e6-885f-ae644aae0b6f.png)

2) **Run this command:**
sudo touch /private/var/db/.AccessibilityAPIEnabled

3) **Restart the pokemongo-webspoof.app.**

**Note:**
Worked for me on Sierra/iOS10

**Comment by Gernatch** [comment]
FYI. Not sure if step 1 is absolutely required. I disabled it at one point and the errors came back. Leaving mine checked.

**Comment by dickverbunt** [comment]
Didn't work for me.

**Comment by Gernatch** [comment]
That's odd. Worked on 2 computers for me. However when I tried running the script alone while the game was running, I still got the error. Are you getting the error in the game or just in the script?

**Comment by dickverbunt** [comment]
The error in map screen of pokemongo-webspoof. Also when I try to run it from the script editor I get the same error.
